blocks/top-table-list-block/features/top-table-list/default.scss

```diff
@@ -1,120 +1,116 @@
-.top-table-list-container {
-  .top-table-list-section-small {
-    & > .small-promo {
-      img {
-        width: 100%;
-      }
-
-      .promo-container {
-        display: flex;
-        flex-direction: row;
+.top-table-list-section-small {
+  hr {
+    width: 100%;
+  }
+}
 
-        & > div {
-          &:nth-child(odd) {
-            padding-right: 0.75rem;
-          }
-      
-          &:nth-child(even) {
-            padding-left: 0.75rem;
-          }
-        }
+.top-table-list-small-promo {
+  img {
+    width: 100%;
+  }
 
-        &.horizontal.reverse {
-          flex-direction: row-reverse;
+  .promo-container {
+    display: flex;
+    flex-direction: row;
 
-          & > div {
-            &:nth-child(odd) {
-              padding-left: 0.75rem;
-              padding-right: 0;
-            }
-        
-            &:nth-child(even) {
-              padding-left: 0;
-              padding-right: 0.75rem;
-            }
-          }
-        }
+    > div {
+      &:nth-child(odd) {
+        padding-right: 0.75rem;
+      }
 
-        &.vertical {
-          @media only screen and (min-width: map-get($grid-breakpoints, 'md')) {
-            flex-direction: column;
+      &:nth-child(even) {
+        padding-left: 0.75rem;
+      }
+    }
 
-            & > div {
-              padding: 0;
+    &.horizontal.reverse {
+      flex-direction: row-reverse;
 
-              &:first-child {
-                padding-bottom: 0.75rem;
-              }
-            }
+      > div {
+        padding-left: 0;
+        padding-right: 0;
 
-            &.reverse {
-              flex-direction: column-reverse;
+        &:nth-child(odd) {
+          padding-left: 0.75rem;
+        }
 
-              & > div {
-                padding: 0;
-  
-                &:last-child {
-                  padding-bottom: 0.75rem;
-                }
-              }
-            }
-          }
+        &:nth-child(even) {
+          padding-right: 0.75rem;
         }
+      }
+    }
 
-        .promo-headline {
-          flex: 2;
-
-          &.headline-wrap-vertical {
-            height: 4.6em;
-            overflow: hidden;
-            display: -webkit-box;
-            -webkit-box-orient: vertical;
-            -webkit-line-clamp: 3;
-            -moz-box-oriented: vertical;
-            /* opera */
-            text-overflow: -o-ellipsis-lastline;
-          }
+    &.vertical {
+      @media only screen and (min-width: map-get($grid-breakpoints, 'md')) {
+        flex-direction: column;
+
+        > div {
+          padding: 0;
 
-          &.headline-wrap-horizontal {
```

Comment:
@JackHowa This was the main issue - there was extra height being set for horizontal, added with the padding bottom which I changed to be margin bottom - line 51